### DISCUSSION TITLE
Regarding Ensemble Restart file

### DISCUSSION THREAD

**Discussion by Gayathridevi123** [NASA-LIS/LISF#1309] [Q&A] [answered]
Hi, 

I would like to clarify about ensemble restart file. I'm trying out the DA simulation for 1/1/2015 - 1/1/2019. So I generated an ENRST file for DA using the open-loop restart file (LIS_RST_NOAH39_201901010000.d01). And in the ENRST file, the time is showing:- time:1, 2019-01-01. I would like to know if this ENRST file contains all the initial conditions that need to start from a 2015 yr DA simulation as the time is showing only once (2019-01-01).  

Thank you
Gayathri

**Comment by dmocko** [comment]
Hi @Gayathridevi123 

The ENRST file that you generated represents the model states on 2019-01-01, as you state.

How did you initialize your open-loop?  I am assuming you started it on 2015-01-01, correct?

**Reply by Gayathridevi123** [reply]
Hi @dmocko,

Thank you for your reply.

Yeah for open loop I performed a spin up and I started the simulation on 2015-01-01.

So for doing DA assimilation for 2015 to 2019 how I need to initialise it using ensemble restart file.

**Comment by dmocko** [comment, marked as answer]
OK, then I recommend that you use the same RST file for your DA initialization that you used for your open loop.

I am guessing that after your spin-up, the RST file that you used to initialize your open loop run was LIS_RST_NOAH39_201901010000.d01.  Yes, make an ensemble restart file out of this file, as use it to initialize your DA run.

The point is that your Open Loop run should have the same initial states at your DA run.  By making the RST file an ensemble file, you are just setting those single-instance initial states onto all of your ensemble members.

**Reply by Gayathridevi123** [reply]
Thank you @dmocko for your suggestion.  I like to clarify about the lis public test case also.. For test case step 2 it is performing open loop simulation for 2017 year and in the end it creates 2018 Jan 1 rst file. In step 3 ensemble restart file is generated using the 2018 Jan 1 open loop restart file. This 2018 Jan 1 ensemble restart file is giving for 2017 DA simulation in step 6. Can you please check and let me know about this?

**Reply by dmocko** [reply]
That is just an example for our functional test case, thanks.

**Reply by Gayathridevi123** [reply]
Ok thank you
